pkg/fanal/analyzer/language/python/packaging/packaging.go

```diff
@@ -98,12 +194,13 @@ func (a packagingAnalyzer) open(file *zip.File) (dio.ReadSeekerAt, error) {
 }
 
 func (a packagingAnalyzer) Required(filePath string, _ os.FileInfo) bool {
-	for _, r := range requiredFiles {
-		if strings.HasSuffix(filePath, r) {
-			return true
-		}
-	}
-	return false
+	return strings.Contains(filePath, ".dist-info") || required(filePath)
```

Comment:
do we need all files from `.dist-info`?
licenseclassifier consumes a lot of resurces.  I am not sure that using it for all files is good idea.

Reply:
@DmitriyLewen I am adding all the files, since it is not known in advance which of the files is the license file. No need to worry about performance, only 1 file is scanned, which is specified in the `License-File` field.

Reply:
oh, right! Thanks!